src/pages/en/guides/publish-to-npm.md

```diff
@@ -10,6 +10,8 @@ Publishing an Astro component is a great way to reuse your existing work across
 
 Looking for inspiration? Check out some of [our favorite themes & components](/en/themes) from the Astro community. You can also [search npm](https://www.npmjs.com/search?q=keywords:astro-component) to see the entire public catalog.
 
+> Don't want to go it alone? Check out [Astro Community's component template](https://github.com/astro-community/component-template) for a community-supported, out-of-the-box template!
```

Comment:
do* :smile:

Reply:
D'oh! Did I miss the intention here? I was thinking of playing up the "community support" angle by "not being alone making a package... use the community template"

Is it more accurate to say that using the Astro Community template is like... going off on your own??

Reply:
> Don't want to go it alone?

I was talking about this sentence haha!

Reply:
This looks really good, apologies for not including this before @Princesseuh it totally slipt my mind